src/main/java/com/redhat/cloud/notifications/db/ApplicationResources.java

```diff
@@ -92,9 +84,13 @@
 
     public Multi<EventType> getEventTypes(UUID appId) {
```

Comment:
This probably has been changed already in master to `Uni<List<EventType>>`, but leaving a note in case is not.

Reply:
This one is still a `Multi` on master. I don't remember why, but this is one of the few cases where we really needed the `Multi`.

Reply:
We should leave a note that we require the `Multi` there and the reason if possible.

Reply:
I think we're using too many `Multi` in the app while we rarely need them. `Multi` is initially supposed to be used to stream items, not to return and/or transform a `List` of items that is completely available at the beginning of the code execution. That could be the subject of a dedicated PR to clean the code.

Reply:
+1

We should clean and ensure we are following some guidelines of when to use Uni/Multi vs stream